test/clojure/core_test/eq.cljc

```diff
@@ -0,0 +1,116 @@
+(ns clojure.core-test.eq
+  (:require [clojure.test :as t]))
+
+;; Doesn't test (yet):
+;; * Java collections for JVM Clojure
+;; * clojure.lang.PersistentQueue
+;; * ##Inf and some other special forms
+;; * Records
+
+(t/deftest two-scalars
```

Comment:
Please follow the existing tests. Here's a reference: https://github.com/jank-lang/clojure-test-suite/blob/main/test/clojure/core_test/inc.cljc

1. Wrap everything in `when-var-exists`
2. Use one `deftest` with multiple `testing` forms within

You can accomplish both of these by using our babashka task for creating tests! Just run

```bash
bb new-test =
```

Reply:
I manually migrated my code over to this format.